src/domain/entities/GizmoEntityItem.ts

```diff
@@ -0,0 +1,63 @@
+//
+//  GizmoEntityItem.ts
+//
+//  Created by Julien Merzoug on 19 Jul 2022.
+//  Copyright 2022 Vircadia contributors.
+//  Copyright 2022 DigiSomni LLC.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+import { CommonEntityProperties } from "../networking/packets/EntityData";
+import PropertyFlags from "../shared/PropertyFlags";
+import { EntityPropertyFlags } from "./EntityPropertyFlags";
+import RingGizmoPropertyGroup from "./RingGizmoPropertyGroup";
+
+
+// WEBRTC TODO: Replace Record<string, never> with GizmoEntityItem's special properties.
+type GizmoEntitySubclassProperties = Record<string, never>;
+
+// WEBRTC TODO: Make a union with GizmoEntitySubclassProperties.
+type GizmoEntityProperties = CommonEntityProperties;
+
+type GizmoEntitySubclassData = {
+    bytesRead: number;
+    properties: GizmoEntitySubclassProperties;
+};
+
+
+class GizmoEntityItem {
+    // C++  class GizmoEntityItem : public EntityItem
+
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
```

Comment:
```suggestion
```
These appear to be not needed. Here and elsewhere.

Reply:
Correct, in `GizmoEntityItem.ts` and `GridEntityItem.ts` it is not needed.

But in the following files `data` is not used:
`BloomPropertyGroup.ts`, `HazePropertyGroup.ts`, `LightEntityItem.ts`, `PulsePropertyGroup.ts` and `RingGizmoPropertyGroup.ts`.